src/bun.js/bindings/webcore/MessagePort.cpp

```diff
@@ -427,8 +451,7 @@ Ref<MessagePort> MessagePort::entangle(ScriptExecutionContext& context, Transfer
 bool MessagePort::addEventListener(const AtomString& eventType, Ref<EventListener>&& listener, const AddEventListenerOptions& options)
 {
     if (eventType == eventNames().messageEvent) {
-        if (listener->isAttribute())
-            start();
```

Comment:
Please see my comment on actual issue. https://github.com/oven-sh/bun/issues/19863#issuecomment-2929201194

I will emphasis that we should make `WebCore` implementation to align withe the JS specs and have that NodeJS compatibility layer abstracted away. 

And until then just having documentation for such points would be suffice.